lib/src/screens/send/widgets/send_card.dart

```diff
@@ -665,22 +669,41 @@ class SendCardState extends State<SendCard> with AutomaticKeepAliveClientMixin<S
     _effectsInstalled = true;
   }
 
-  Future<void> _setTransactionPriority(BuildContext context) async {
+  Future<void> pickTransactionPriority(BuildContext context) async {
     final items = priorityForWalletType(sendViewModel.walletType);
     final selectedItem = items.indexOf(sendViewModel.transactionPriority);
+    final customItem = items.firstWhereOrNull((element) => element == sendViewModel.getBitcoinTransactionPriorityCustom);
```

Comment:
`sendViewModel.getBitcoinTransactionPriorityCustom` will throw an exception for other wallet types

Reply:
I've tested it with other wallet types, it returns just null for customItem

Reply:
will throw null check operator called on a null value on Monero.com, because you are using `bitcoin!.` but i will push a fix for this since it's minor